test/FeeCollector.t.sol

```diff
@@ -0,0 +1,539 @@
+            uint256 preTotalClientBalances = feeCollector.totalClientBalances(token);
+            uint256 preClientFeeBalance = feeCollector.balances(TEST_CLIENT, token);
+
+            // Act: collect fees
+            feeCollector.collectFees(TEST_CLIENT, token, _amount);
+
+            // Post-act balances
+            uint256 postContractBalance = IERC20(token).balanceOf(feeCollectorAddr);
+            uint256 postTotalClientBalances = feeCollector.totalClientBalances(token);
+            uint256 postClientFeeBalance = feeCollector.balances(TEST_CLIENT, token);
+
+            // Assertions
+            assertEq(postContractBalance, preContractBalance + _amount);
+            assertEq(postTotalClientBalances, preTotalClientBalances + clientFee);
+            assertEq(postClientFeeBalance, preClientFeeBalance + clientFee);
+        }
+    }
+
+    /// @dev
+    // - The FeeCollector's token balance should increase by _amount
+    // - The token totalClientBalances should not change
+    // - The above should be true when _client is sent as address(0)
+    function testFuzz_CollectFeesNoClient(uint256 _amount) external payable {
+        for (uint256 i; i < positions.length; i++) {
+            // Test Variables
+            address token = positions[i].cToken;
+
+            // Bound fuzzed variables
+            _amount = bound(_amount, assets.minCAmts(token), assets.maxCAmts(token));
+
+            // Fund positionOwner with _amount of token
+            _fund(positionOwner, token, _amount);
+
+            // Approve FeeCollector contract to spend token
+            IERC20(token).approve(feeCollectorAddr, _amount);
+
+            // Pre-act balances
+            uint256 preContractBalance = IERC20(token).balanceOf(feeCollectorAddr);
+            uint256 preTotalClientBalances = feeCollector.totalClientBalances(token);
+
+            // Act: collect fees
+            feeCollector.collectFees(address(0), token, _amount);
+
+            // Post-act balances
+            uint256 postContractBalance = IERC20(token).balanceOf(feeCollectorAddr);
+            uint256 postTotalClientBalances = feeCollector.totalClientBalances(token);
+
+            // Assertions
+            assertEq(postContractBalance, preContractBalance + _amount);
+            assertEq(postTotalClientBalances, preTotalClientBalances);
+        }
+    }
+
+    /// @dev
+    // - The FeeCollector's cToken balance should increase by protocolFee
+    // - The cToken totalClientBalances should increase by clientFee
+    // - The client's cToken balance on the FeeCollector contract should increase by clientFee
+    function testFuzz_CollectFeesWithClientIntegrated(uint256 _ltv, uint256 _cAmt) external payable {
+        for (uint256 i; i < positions.length; i++) {
+            // Test Variables
+            address positionAddr = positions[i].addr;
+            address cToken = positions[i].cToken;
+
+            // Bound fuzzed variables
+            _ltv = bound(_ltv, 1, 60);
```

Comment:
Let's hard code this to `50`, and remove it as a fuzzed variable, as it has nothing to do with collectFees. It's related to borrowing on Aave.